develop/TOOLS/DIS/Troubleshooting/DIS_Troubleshooting_CompilationFailure.md

```diff
@@ -0,0 +1,32 @@
+---
+uid: DIS_Troubleshooting_CompilationFailure
+keywords: dis publish issue, dis compile issue, dis copy to clipboard issue
+---
+
+# Package creation problems
+
+## Problem
+
+- Publishing a connector or script fails with an exception in DIS.
+
+- Saving a compiled connector or script fails with an exception in DIS.
+
+- *Copy Protocol to Clipboard* fails with an exception in DIS.
+
+## Possible causes
+
+- Invalid configuration.
+
+- No access to internet (when using NuGet packages).
+
+- Syntax that is not covered yet.
+
+## Possible solutions
+
+Check if the configuration is correct:
+
+- Make sure proper XML is used.
+
+- Use the PackageReference package management format (see [Consuming NuGet packages](xref:Consuming_NuGet)).
+
+If this does not solve the problem, it likely involves an unexpected scenario with the solution and the parsing within DIS. Report the issue to the DIS team, making sure to include a copy of the solution and the output logging of Visual Studio. For more info, see [Retrieving information in case a problem occurs](xref:DIS_Troubleshooting_RetrieveInformation).
```

Comment:
Which email address should they contact? It would be more helpful to immediately specify this here.

Reply:
I'm not sure what to put here exactly as internally it will be via UOSA, eventually externally as well. But some users contact DIS team, some contact Data-Acq and some contact Tech Support. 
We're in the process of streamlining it (UOSA) but externally there is no fixed process yet.

Reply:
OK, in that case we can leave this as it is for now, and update it as soon as it's possible via UOSA for all users.